system/Router/RouteCollection.php

```diff
@@ -806,8 +806,9 @@ public function resource(string $name, array $options = null): RouteCollectionIn
 		// In order to allow customization of the route the
 		// resources are sent to, we need to have a new name
 		// to store the values in.
-		$new_name = ucfirst($name);
-
+		//$new_name = ucfirst($name);
+		// Michal thought, modifying a single line in all parts of the namespace
```

Comment:
Please remove these two lines. Changes are tracked by git history so commenting the code is not necessary. Also, mentioning my name in the comment is not worth it :)